XLEngine.conf

```diff
@@ -0,0 +1,10 @@
+[general]
+width = 1920
+height = 1080
+fullscreen = false 
+vsync = false 
+emulate-low-res = false 
+renderer = opengl 
+ 
+[DaggerXL] 
+data-path = 
```

Comment:
Can we add more comments here and set the width and height to something lower? It can still be 16:9 ratio, just not full 1080.
```
[general]
width = 1280 # window/screen width, 320 minimum
height = 720 # window/screen height, 200 minimum
fullscreen = false # true or false
vsync = false # true or false
emulate-low-res = false # true to render at 320x200 and stretch to fit, or false
renderer =  opengl # opengl, soft32 (true-color software), soft8 (paletted software)

[DaggerXL]
data-path = # Full path to game data, e.g. C:\DAGGER\ARENA2

[DarkXL]
data-path = # ...

[BloodXL]
data-path = # ...

[OutlawsXL]
data-path = # ...
```

Reply:
Has the config file been tested on windows?
I've tried to use it but XLEngine reports every line as invalid and crashes shortly after